pkgs/development/python-modules/rainbowstream/requests.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/setup.py b/setup.py
+index b04ce8e..a5347de 100644
+--- a/setup.py
++++ b/setup.py
+@@ -9,7 +9,7 @@ version = '1.3.1'
+ install_requires = [
+     "python-dateutil",
+     "arrow",
+-    "requests==2.5.3",
++    "requests>=2.5.3",
```

Comment:
it would be easier to maintain if you just did `sed -i 's/requests.*"/requests"/' setup.py`